common/lib/common-utils/pnpm-lock.yaml

```diff
@@ -584,13 +581,15 @@ packages:
   '@humanwhocodes/config-array@0.11.14':
     resolution: {integrity: sha512-3T8LkOmg45BV5FICb15QQMsyUSWrQ8AygVfC7ZG32zOalnqrilm018ZVCw0eapXux8FtA33q8PSRSstjee3jSg==}
     engines: {node: '>=10.10.0'}
+    deprecated: Use @eslint/config-array instead
```

Comment:
Looks like this file somehow got missed in https://github.com/microsoft/FluidFramework/pull/24385 . pnpm doesn't seem to make this change when updating on main, but does on my branch with these changes. Odd.

I supposed its fine for these to be included here, but I don't know why pnpm would make these changes on this branch and not the other.